sdk/core/core-rest-pipeline/src/util/tokenCycler.ts

```diff
@@ -124,11 +127,13 @@ export function createTokenCycler(
     ...tokenCyclerOptions
   };
 
+  const errorMessage = "Failed to refresh access token.";
+
   /**
    * This little holder defines several predicates that we use to construct
    * the rules of refreshing the token.
    */
-  const cycler = {
+  const cyclerState = {
```

Comment:
I'm not too picky about this, but the reason I avoided calling this `state` is because it doesn't actually encapsulate the state of the cycler, which is actually stored in `token` and `refreshWorker`. This object simply represents a structured way to query the state information through readonly computed properties.

If I rewrote this today, I might've called it `self` instead of `cycler`, since I think that `if (self.mustRefresh)` has a nice ring to it.

Reply:
I like `self.mustRefresh`!